.readthedocs.yml

```diff
@@ -10,6 +10,7 @@ build:
     python: "3.12"
   apt_packages:
     - cmake
+    - ccache
```

Comment:
This is a great idea, but it will likely only yield a significantly improved performance if we maintain the cache across builds. That means uploading it somewhere and restoring it in each new build instance. RTD may have a mechanism for storing and restoring build artifacts.